datadog_checks_downloader/datadog_checks/downloader/parameters.py

```diff
@@ -28,7 +28,7 @@ def substitute(target_relpath):
     if not wheel_distribution_name.startswith('datadog_'):
         raise NonDatadogPackage(wheel_distribution_name)
 
-    standard_distribution_name = safe_name(wheel_distribution_name)
+    standard_distribution_name = canonicalize_name(wheel_distribution_name)
```

Comment:
@ofek have you checked that this function does the same thing? I assume you have, but just double-checking.

Reply:
For our use case, yes